packages/amplify_auth_plugin_interface/lib/src/SignIn/AuthNextSignInStep.dart

```diff
@@ -18,5 +18,5 @@ import '../types/AuthNextStep.dart';
 
 class AuthNextSignInStep extends AuthNextStep {
   String signInStep;
-  AuthNextSignInStep({additionalInfo, @required codeDeliveryDetails, @required this.signInStep}) : super(additionalInfo: additionalInfo, codeDeliveryDetails: codeDeliveryDetails);
+  AuthNextSignInStep({required additionalInfo, required codeDeliveryDetails, required this.signInStep}) : super(additionalInfo: additionalInfo, codeDeliveryDetails: codeDeliveryDetails);
```

Comment:
Why is it necessary to make `additionalInfo` required?

Reply:
You're right it's not required as AuthNextStep provides a default value.